### ISSUE TITLE
Fail to reproduce

### ISSUE THREAD

**Issue by tirrorex** [azmeuk/FoobarLib#2]
Hello, just want to check something quickly.
I can see that firebase is not included in s.dependency and you also didn't include it in your podfile.
Does that mean that you added the framework manually to your project?

**Comment by azmeuk** [comment]
Yes, [there](https://github.com/azmeuk/FoobarLib/tree/master/FoobarLib/Frameworks/Firebase) :)

**Comment by tirrorex** [comment]
So just to be clear, it is not possible to do this while adding firebase framework with cocoapods?
Or is this due to other reasons? (if so i would like to know them)
I've been struggling with this for two days straight :/
